package.json

```diff
@@ -11,6 +11,7 @@
     "refappSelenium": "mvn test -Dcucumber.filter.tags='@selenium and not @initialSetup'",
     "refapp2VitalsAndTriagingTest": "mvn test -Dcucumber.filter.tags='@selenium and @vitals'",
     "refapp2ClinicalVisitTest": "mvn test -Dcucumber.filter.tags='@selenium and @clinicalVisit'",
+ 	"refapp2InternalPatientTransferTest":"mvn test -Dcucumber.filter.tags='@selenium and @internalTransfer'",
```

Comment:
Put this on the same line.Thanks.

Reply:
I have tried this like 8 times and I don't seem to get the exact spacing. On my machines it shows it's well aligned but here it's a different case. I there some sort of linter or formatter that you are using?